src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.service;
+
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String mail, String password);
+
+    User register(String email, String password);
+}
```

Comment:
your login method should throw an exception

public interface AuthenticationService {
    User login(String email, String password) throws AuthenticationException;
    User register(String email, String password);